lib/Graph/Graph.cpp

```diff
@@ -684,8 +684,7 @@ Node *Function::createChannelShuffle(llvm::StringRef name, NodeValue input,
 
 Node *Function::createSqueeze(llvm::StringRef name, NodeValue input,
                               llvm::ArrayRef<size_t> axes) {
-  auto originalSize = axes.size();
-  assert(originalSize > 0 && "Parameter `axes` must be provided.");
+  assert(axes.size() > 0 && "Parameter `axes` must be provided.");
```

Comment:
since there is already a method `empty()` for an ArrayRef I'd use that instead of size() > 0.